README.md

```diff
@@ -501,8 +501,66 @@ container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity
     'biocontainers/fastqc:0.11.9--0' }"
 ```
 
+For IRIDA Next Nextflow pipelines running in Azure and using Docker, the default container registry should be set to the publicly available `quay.io` registry in the `nextflow.config` file using the `docker.registry` parameter.
```

Comment:
Should we state that 'quay.io' by default follows nf-core convention [here](https://nf-co.re/docs/usage/getting_started/configuration#docker-registries)

Reply:
Done here: [2f0ba43](https://github.com/phac-nml/pipeline-standards/pull/24/commits/2f0ba43b3d0216d42e906e2c168c10197436342d)